tools/wpt/run.py

```diff
@@ -338,27 +338,33 @@ class EdgeChromium(BrowserSetup):
     browser_cls = browser.EdgeChromium
 
     def setup_kwargs(self, kwargs):
+        browser_channel = kwargs["browser_channel"]
+        if kwargs["binary"] is None:
+            binary = self.browser.find_binary(channel=browser_channel)
+            if binary:
+                kwargs["binary"] = self.browser.find_binary()
+            else:
+                raise WptrunError("Unable to locate Edge binary")
         if kwargs["webdriver_binary"] is None:
             webdriver_binary = self.browser.find_webdriver()
 
-            if webdriver_binary is None:
+            # Install browser if none are found or if it's found in venv path
+            if webdriver_binary is None or self.venv.bin_path in webdriver_binary:
```

Comment:
```suggestion
            if webdriver_binary is None or webdriver_binary in self.venv.bin_path:
```
Pretty sure this is inverted.